homeassistant/components/xiaomi_miio/humidifier.py

```diff
@@ -180,7 +180,7 @@ def __init__(self, name, device, entry, unique_id, coordinator):
             self._min_humidity = 30
             self._max_humidity = 80
             self._humidity_steps = 10
-        elif self._model in [MODEL_AIRHUMIDIFIER_CA4]:
+        elif self._model in (MODEL_AIRHUMIDIFIER_CA4,):
```

Comment:
Maybe better:
```python
elif self._model == MODEL_AIRHUMIDIFIER_CA4:
```

Reply:
@bieniu The focus for this PR was to only replace lists with tuples. It doesn't really make sense to change one or two more these into equality checks. I might consider adding another pylint check for it to replace all of them at once.